Makefile

```diff
@@ -1,10 +1,11 @@
 PREFIX ?= $(shell pwd)
+FILES ?= $(shell find . -type f -name '*.go' -not -path "./vendor/*")
```

Comment:
Ah, actually `go list ./...` should work and exclude `vendor/` as of go1.9 by default.

Reply:
Yes, but the issue is that goimports needs relative path.. not package relative to GOPATH